{{cookiecutter.project_name}}/{{cookiecutter.package_name}}/version.py

```diff
@@ -8,5 +8,5 @@
 This module contains project version information.
 """
 
-__version__ = '0.0.1'  #: the working version
-__release__ = '0.0.1'  #: the release version
+__version__ = "0.0.1"  #: the working version
```

Comment:
I think this is a mistake that's always been present so we wouldn't need to deal with it here... but now that I'm looking at it we should probably be replacing `"0.0.1"` with `"{{cookiecutter.project_version}}"`.

Reply:
This has been addressed. Not too sure if making release==version is good :thinking: